.babelrc

```diff
@@ -1,5 +1,16 @@
 {
   "presets": [
     "latest"
-  ]
+  ],
+  "env": {
+    "production": {
+      "presets": [
+        ["latest", {
+          "es2015": {
+            "modules": false
+          }
+        }]
+      ]
+    }
+  }
 }
```

Comment:
ES2015 modules can be parsed by webpack 2. This enables tree shaking in the production build.

Reply:
I've also switched to babel-preset-env since it replaced babel-preset-latest.

Reply:
You'll need to update all "latest" with "env" in .babelrc.

Reply:
@jcscott2015 - Correct, and I did that as you can see above.